osl/interface_linux.go

```diff
@@ -19,17 +19,18 @@ import (
 type IfaceOption func(i *nwIface)
 
 type nwIface struct {
-	srcName     string
-	dstName     string
-	master      string
-	dstMaster   string
-	mac         net.HardwareAddr
-	address     *net.IPNet
-	addressIPv6 *net.IPNet
-	llAddrs     []*net.IPNet
-	routes      []*net.IPNet
-	bridge      bool
-	ns          *networkNamespace
+	srcName        string
+	dstName        string
+	master         string
+	dstMaster      string
+	mac            net.HardwareAddr
+	address        *net.IPNet
+	addressIPv6    *net.IPNet
+	secondaryAddrs []*net.IPNet
```

Comment:
Can you please rename to `ipAliases`.
It is more inline with the linux terminology and it will avoid the indentation changes.